exchanges/mock/server_test.go

```diff
@@ -59,7 +76,17 @@ func TestNewVCRServer(t *testing.T) {
 		"http://localhost:300/somethingElse?"+queryString,
 		nil,
 		bytes.NewBufferString(""), true)
-	assert.ErrorContains(t, err, "connection refused", "SendHTTPRequest should return the correct error")
+	assert.Error(t, err, "SendHTTPRequest should return the correct error")
```

Comment:
```
	var netErr *net.OpError
	assert.ErrorAs(t, err, &netErr, "SendHTTPRequest should return a net.OpError for an invalid host")
```
	
I reckon something like this is sufficient due to different messages between different OS's